src/SearchField/SearchField.doc.js

```diff
@@ -23,6 +23,10 @@ card(
         type: 'string',
         required: true,
       },
+      {
+        name: 'onBlur',
+        type: '({ value: string }) => void',
```

Comment:
Do we need to pass the value in here? It hasn't changed with the event and should be the same as what's held by the parent.